operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -147,6 +168,8 @@ protected Status createStatus() {
                 .onComplete(context.failing(v -> context.verify(() -> {
                     MeterRegistry registry = metrics.meterRegistry();
 
+                    assertThat(registry.getMeters().get(0).getId().getTags().size(), is(2));
+                    assertThat(registry.getMeters().get(0).getId().getTags().get(1), is(Tag.of("selector", "")));
```

Comment:
Should we apply the same logic here as we did above? I.e. test it for different selectors? Or if we think that it is not needed to test the selector here anymore, do we really need these asserts?

Plus the same as above ... the `get(0)` looks weird. Can't we get the metric by name?

This applies also to the same tests below.

Reply:
for failing reconcile I tried using a non empty label but seem like it breaks the unit test, I get some sort of async error

Reply:
But I guess the label should be there even in that case, or? So it probably needs to be addressed one way or another.

Reply:
```java.lang.AssertionError: java.lang.AssertionError: The asynchronous result was expected to have failed``` i get this error when i run it with valued selector
Empty labels work fine

Reply:
I guess that either means there is some error in the code or in the test.